libp2p/NodeTable.cpp

```diff
@@ -304,101 +308,117 @@ vector<shared_ptr<NodeEntry>> NodeTable::nearestNodeEntries(NodeID _target)
     return ret;
 }
 
-void NodeTable::ping(NodeEntry const& _nodeEntry, boost::optional<NodeID> const& _replacementNodeID)
+void NodeTable::ping(Node const& _node, shared_ptr<NodeEntry> _replacementNodeEntry)
+{
+    // Don't sent Ping if one is already sent
+    if (contains(m_sentPings, _node.id))
+    {
+        LOG(m_logger) << "Ignoring request to ping " << _node << ", because it's already pinged";
```

Comment:
This is being logged quite a lot I should say

Reply:
Yea, I think we benefit from more logging since it makes it easier to track what's going on and detect bugs but wading through the log spew can be challenging. It would be nice to eventually have some additional log levels so we could have more targeted logging.

Reply:
Separating logs into INFO and DEBUG levels would be good. In INFO only actual changes (node added, node replaced by) and/or stats (X new nodes discovered).
The rest should go to DEBUG. Later we can consider splitting DEBUG into DEBUG and TRACE.

For aleth-bootnode the networking INFO level should be enabled by default.
For aleth all networking logs should be disabled.

Reply:
> For aleth-bootnode the networking INFO level should be enabled by default.

Agreed, filed #5499 to track this